src/nomad_material_processing/general.py

```diff
@@ -449,36 +449,71 @@ class Miscut(ArchiveSection):
     The miscut in a crystalline substrate refers to
     the intentional deviation from a specific crystallographic orientation,
     commonly expressed as the angular displacement of a crystal plane.
+
+    It can be either directed along one of the crystallographic axes or in a specific
+    direction in the crystal defined by a main direction and the perpendicular one.
+
+    Refs.
+    https://dictionary.iucr.org/Miller_indices
+    https://dictionary.iucr.org/Direction_indices
+
+
     """
 
     angle = Quantity(
         type=float,
         description="""
-        The angular displacement from the crystallographic orientation of the substrate.
+        The miscut angle (or offcut angle, or angular displacement offset) toward
+        the crystallographic orientation of the substrate.
         """,
         a_eln=ELNAnnotation(
             component=ELNComponentEnum.NumberEditQuantity,
             defaultDisplayUnit='deg',
-            label='Miscut Angle',
         ),
         unit='deg',
     )
     angle_deviation = Quantity(
         type=float,
-        description='The ± deviation in the angular displacement.',
+        description='The ± deviation of the angular displacement offset.',
         a_eln=ELNAnnotation(
             component=ELNComponentEnum.NumberEditQuantity,
             defaultDisplayUnit='deg',
-            label='± Miscut Angle Deviation',
+            label='± Angle Deviation',
         ),
         unit='deg',
     )
-    orientation = Quantity(
+    hkl_reciprocal = Quantity(
+        type=str,
+        description="""Orientation of the miscut (or offcut).
+        The reciprocal lattice vector associated with the family of lattice planes is
+        OH = h a* + k b* + l c*, where a*, b*, c* are the reciprocal lattice basis
+        vectors. OH is perpendicular to the family of lattice planes and OH = 1/d where
+        d is the lattice spacing of the family.""",
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+            label='Miscut Reciprocal Indices (hkl)',
+        ),
+    )
+    hkl_direct = Quantity(
```

Comment:
Where did you get the term "direct" from? Is this short for "direction" from: 
"In the context of crystal **directions** (not planes), the corresponding notations are:
 - [ h k ℓ ] , with square instead of round brackets, denotes a direction in the basis of the direct lattice vectors instead of the reciprocal lattice; and
- similarly, the notation ⟨ h k ℓ ⟩ denotes the set of all directions that are equivalent to [ h k ℓ ] by symmetry."

from this wikipedia page: https://en.wikipedia.org/wiki/Miller_index ?
If so, I would change the property name to `hkl_direction`.